### PR TITLE
Changing module reference links

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Changing terraform module reference links.

### CODE REVIEW

**Review by patheard — CHANGES_REQUESTED** [review]
This is downgrading one of the databases auto-upgrades.  You can fix it by matching the [version here in the RDS cluster resource](https://github.com/cds-snc/share-files-securely/blob/a775a14acb682ab8265805f1b9118a041ca2d043/terraform/rds.tf#L11).

**Comment by sylviamclaughlin** [comment]
> This is downgrading one of the database's auto-upgrades. You can fix it by matching the [version here in the RDS cluster resource](https://github.com/cds-snc/share-files-securely/blob/a775a14acb682ab8265805f1b9118a041ca2d043/terraform/rds.tf#L11).

Thanks for noticing this Pat! I have updated the version in the rds file but let me know if I have misunderstood you. Thanks again!

File: terraform/file_scanning.tf [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 
-  product_name          = "share-files-securely"
   s3_upload_bucket_name = module.share_files_securely_bucket.s3_bucket_id
 
   billing_tag_value = var.billing_code
 }
 
 module "s3_scan_objects_dev" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
+  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.1"
```
Comment (patheard) [inline terraform/file_scanning.tf:10]: Could you bump this to the latest and use the new input I just added.  This PR made me realize the module had a bug that would only let it be used once per account 😅 
```suggestion
  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.5"
```

File: terraform/file_scanning.tf [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 
-  product_name                  = "share-files-securely-dev"
```
Comment (patheard) [inline terraform/file_scanning.tf:13]: You can use the new suffix here so things install properly:
```hcl
scan_queue_suffix = "DEV"
```

Comment (sylviamclaughlin) [reply terraform/file_scanning.tf:13]: Done! Thank you for making the change to the base module and for noticing this.

**Review by patheard — APPROVED** [review]

**Pull request merged**